README.md

```diff
@@ -77,6 +78,17 @@
     php bin/console cache:clear
     ```
 
+## Add anonymization configuration
+
+ ```yaml
+synolia_sylius_gdpr:
+    disable_default_mappings: false # False by default
+    anonymization:
+        mappings:
+            paths:
+                - # Your\Paths
```

Comment:
How about adding an example of the file content? Or to put a link to an example file?